### PR TITLE
Update README with "purpose" statement

### LINKED ISSUE #70: Help wanted: Own tests or promises-tests tests?
I am writing a Promises A+ implementation (just for fun and in order to understand promises better), but I don't understand what is the purpose of writing my own tests _VS_ using `promises-aplus/promises-tests` or even both. I see some implementations write their own tests, but isn't it enough to make `promises-tests` pass? I would like to know if I am missing something. 

Cheers.

### PR DESCRIPTION
**Pull request by briancavalier**
Any tweaks or suggestions?

Close #70

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ This suite tests compliance of a promise implementation with the [Promises/A+ sp
 
 [Promises/A+ specification]: https://github.com/promises-aplus/promises-spec
 
+Passing the tests in this repo means that you have a Promises/A+ compliant implementation of the `then()` method, and you can display the Promises/A+ logo in your README. You can also send a PR to have your impl listed on the [implementations page](https://promisesaplus.com/implementations).
```
Comment (domenic) [inline README.md]: s/PR/pull request, and maybe link to the main repo.

s/impl/implementation

Otherwise LGTM!

**Comment by briancavalier** [comment]
Great suggestions.  Incorporated and force pushed.

**Comment by briancavalier** [comment]
Ping @domenic.  This seems good to go, but wanted to double check since it's been around a while.

**Comment by domenic** [comment]
Oh, thanks for the ping! LGTM.

**Comment by briancavalier** [comment]
Cool, thanks!

**Pull request merged**